src/main/kotlin/org/jitsi/jicofo/bridge/ExternalBridgeSelectionStrategyConfig.kt

```diff
@@ -0,0 +1,29 @@
+package org.jitsi.jicofo.bridge
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.optionalconfig
+import java.time.Duration
+
+class ExternalBridgeSelectionStrategyConfig private constructor() {
+    val url: String? by optionalconfig {
+        "${BASE}.url".from(JitsiConfig.newConfig)
+    }
+    fun url() = url
+
+    val timeout: Duration? by optionalconfig {
+        "${BASE}.timeout".from(JitsiConfig.newConfig)
+    }
+    fun timeout() = timeout
+
+    val fallbackStrategy: String? by optionalconfig {
```

Comment:
Perhaps it's better to make this required (`by config`) and provide a good default value?